lib/active_interaction/filters/hash_filter.rb

```diff
@@ -77,7 +77,7 @@ def convert(value)
 
     # rubocop:disable Style/MissingRespondToMissing
     def method_missing(*args, &block)
-      super(*args) do |klass, names, options|
+      super do |klass, names, options|
```

Comment:
I ran `rake` and rubocop complained about this one, although it seems unrelated to any of my changes, not sure why it wouldn't have come up before 🤔

Reply:
I can look into this.